pkg/auth/model/model.go

```diff
@@ -41,23 +41,20 @@ type User struct {
 }
 
 // hashPassword generates a hashed password from a plaintext string
```

Comment:
1. Move the comment to the function
2. The function is now exposed so start with upper case

Reply:
moved and was already uppercase.